integration/src/main/kotlin/kotlinx/benchmark/integration/BenchmarkConfiguration.kt

```diff
@@ -0,0 +1,25 @@
+package kotlinx.benchmark.integration
+
+class BenchmarkConfiguration {
```

Comment:
Is it possible to reuse the existing class `kotlinx.benchmark.gradle.BenchmarkConfiguration`?

Reply:
Hm, will check it, but anyway, we need to put those values into `build.gradle` inside temp project used for testing.

Reply:
On current moment, plugin is not in classpath of integration module, because `plugin` is included build (to use current version of plugin in examples).
We can add it to classpath in some way, but now `kotlinx.benchmark.gradle.BenchmarkConfiguration` depends on `BenchmarksExtension`, looks like we need to remove this dependency to reuse it.

I have a question, why is included build is used at all? Maybe it's possible to make it as an ordinary module, and after that integration module will be moved just to `integrationTest` source set under it (the same way as it's done in `kotlinx-atomicfu` and `binary-compatibility-validator` gradle plugins)

Reply:
> why is included build is used at all?

Seems like for historical reasons. Let's explore the possibility of making the plugin an ordinal module in a separate effort.